base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -326,12 +315,42 @@ impl ConsensusConstants {
             max_block_transaction_weight: 19500,
             median_timestamp_count: 11,
             emission_initial: 10_000_000.into(),
-            emission_decay: 0.999,
-            emission_decay_int: &EMISSION_DECAY,
+            emission_decay: &EMISSION_DECAY,
             emission_tail: 100.into(),
             max_randomx_seed_height: std::u64::MAX,
-            genesis_coinbase_value_offset: 0.into(),
             proof_of_work: algos,
+            faucet_value: 0 * T,
+        }]
+    }
+
+    pub fn ridcully() -> Vec<Self> {
+        let difficulty_block_window = 90;
+        let mut algos = HashMap::new();
+        // seting sha3/monero to 40/60 split
```

Comment:
I think these values should be based on a simple calculation to show where they come from, as it would make future code maintenance a lot easier, i.e.
``` rust
target_block_interval = 120

// Algo 1 - Blake
block_percentage = 40
target_time = target_block_interval * 100 / block_percentage // 300
max_target_time = target_time * 6 // 1800

// Algo 2 - Monero
block_percentage = 60
target_time = target_block_interval * 100 / block_percentage // 200
max_target_time = target_time * 6 // 1200
```